src/prob/tp_pf_lm.jl

```diff
@@ -0,0 +1,101 @@
+export run_ac_tp_pf_lm, run_tp_pf_lm
+
+
+""
+function run_ac_tp_pf_lm(file, solver; kwargs...)
+    return run_tp_pf_lm(file, PMs.ACPPowerModel, solver; kwargs...)
+end
+
+
+""
+function run_dc_tp_opf_lm(file, solver; kwargs...)
+    return run_tp_pf_lm(file, PMs.DCPPowerModel, solver; kwargs...)
+end
+
+
+""
+function run_tp_pf_lm(data::Dict{String,Any}, model_constructor, solver; kwargs...)
+    return PMs.run_generic_model(data, model_constructor, solver, post_tp_pf; multiconductor=true, kwargs...)
+end
+
+
+""
+function run_tp_pf_lm(file::String, model_constructor, solver; kwargs...)
+    data = ThreePhasePowerModels.parse_file(file)
+    return PMs.run_generic_model(data, model_constructor, solver, post_tp_pf; multiconductor=true, kwargs...)
+end
+
+
+""
+function post_tp_pf_lm(pm::PMs.GenericPowerModel)
+    add_arcs_trans!(pm)
```

Comment:
Add a maintenance issue, `add_arcs_trans!`, should be added as `ref_extention` rather than being called in post_*.  In fact we may consider a more TPPM extension of run_* that includes this.